documentation/code-gen/configure-python-sdk.md

```diff
@@ -0,0 +1,229 @@
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+package-name: azure-mgmt-appconfiguration
+no-namespace-folders: true
+package-version: 0.1.0
+```
+
+``` yaml $(python-mode) == 'update'
+// --------------- For track1 -----------------
+python:                                     
+  no-namespace-folders: true
+  output-folder: $(python-sdks-folder)/appconfiguration/azure-mgmt-appconfiguration/azure/mgmt/appconfiguration
+
+// --------------- For track2 -----------------
+no-namespace-folders: true
+output-folder: $(python-sdks-folder)/appconfiguration/azure-mgmt-appconfiguration/azure/mgmt/appconfiguration
+```
+``` yaml $(python-mode) == 'create'
+python:
+  basic-setup-py: true
+  output-folder: $(python-sdks-folder)/appconfiguration/azure-mgmt-appconfiguration
+basic-setup-py: true
+output-folder: $(python-sdks-folder)/appconfiguration/azure-mgmt-appconfiguration
+```
+~~~
+
+## Multi-api
+If a multi-api package is need to be released, 'batch' should be used in the readme.python.md.
+
+### batch
+The batch is a tag list which are used in the multi-api package. For example:
+~~~
+// file: readme.python.md
+
+### Python multi-api
+
+Generate all API versions currently shipped for this package
+
+```yaml $(multiapi) && !$(track2)           // For track1
+batch:
+  - tag: package-2020-06-01-only
+  - tag: package-2020-05-01-only
+```
+
+```yaml $(multiapi) && $(track2)            // For track2
+clear-output-folder: true
+batch:
+  - tag: package-2020-06-01-only
+  - tag: package-2020-05-01-only
+  - multiapiscript: true
+```
+
+``` yaml $(multiapiscript)                  // For track2
+output-folder: $(python-sdks-folder)/compute/azure-mgmt-compute/azure/mgmt/compute/
+clear-output-folder: false
+perform-load: false
+```
+~~~
+
+### tags of batch
+Then, the output folder and namespace should be configured for each of the tag. For example:
+~~~
+// file: readme.python.md
+
+### Tag: package-2020-06-01-only and python
+
+These settings apply only when `--tag=package-2020-06-01-only --python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+
+``` yaml $(tag) == 'package-2020-06-01-only'
+namespace: azure.mgmt.compute.v2020_06_01
+output-folder: $(python-sdks-folder)/compute/azure-mgmt-compute/azure/mgmt/compute/v2020_06_01
+python:
+  namespace: azure.mgmt.compute.v2020_06_01
+  output-folder: $(python-sdks-folder)/compute/azure-mgmt-compute/azure/mgmt/compute/v2020_06_01
+```
+~~~
+
+
+### multi-api init script
+For track1, multi-api init script need to be configured in readme.md. For example:
+
+~~~
+// file: readme.md
+
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
+    after_scripts:
+      - python ./scripts/multiapi_init_gen.py azure-mgmt-compute
+~~~
+
+
+## Run codegen
+After configure all the readme files, autorest can be used to generate SDK.
+### Track1
```

Comment:
Describe Track 1 (Based on AutoRest version xxxxx that's going to be replaced by version yyyyy)